src/candid_rpc/cketh_conversion.rs

```diff
@@ -186,10 +186,5 @@ pub(super) fn into_hash(value: Hex32) -> Hash {
 }
 
 fn from_address(value: ic_ethereum_types::Address) -> evm_rpc_types::Hex20 {
```

Comment:
Is there a reason to keep this function around?   :slightly_smiling_face:

Reply:
not particularly, I inlined it.